onlinejudge_verify/main.py

```diff
@@ -87,6 +90,52 @@ def subcommand_init() -> None:
         fh.write(verify_yml.replace(b'git+https://github.com/kmyk/online-judge-verify-helper.git@master', b'"online-judge-verify-helper==2.*"'))
 
 
+def subcommand_export(*, path: pathlib.Path, source_dir: pathlib.Path) -> Tuple[str, Set[pathlib.Path]]:
+    dir_path = path.parent
+    if source_dir == pathlib.Path(): source_dir = pathlib.Path(os.getcwd()).resolve()
+
+    # コメントアウトを消す (コメントアウトされた #include を誤認識しないための処理)
+    code_1 = """g++ -I {} -fpreprocessed -dD -E {} | tail -n +2""".format(shlex.quote(str(source_dir)), shlex.quote(str(path)))
+    code_2 = """cat {}""".format(shlex.quote(str(path)))
+    lines_1 = subprocess.check_output(code_1, shell=True).decode().splitlines()
+    lines_2 = subprocess.check_output(code_2, shell=True).decode().splitlines()
+
+    # `#pragma once` と書かれているファイルを記録
+    included_once_files = set()  # type: Set[pathlib.Path]
+    results = []  # type: List[str]
+
+    pragma_once = len(lines_2) > 0 and re.match(r'^#\s*pragma\s+once\s*$', lines_2[0])
+    if pragma_once:
+        included_once_files.add(path.resolve())
+        lines_1, lines_2 = lines_1[1:], lines_2[1:]
+    for line_1, line_2 in zip(lines_1, lines_2):
```

Comment:
```suggestion
    assert len(lines_1) == len(lines_2)
    for line_1, line_2 in zip(lines_1, lines_2):
```

(nitpick) こうやってコメントなし版とコメントあり版を並べるのかしこすぎて逆に「すごいけどこれ本当に動くのか？」と不安になるので `assert` ほしい